AsyncDisplayKit/Details/ASCollectionViewLayoutController.mm

```diff
@@ -164,4 +165,19 @@ - (CGRect)rangeBoundsWithScrollDirection:(ASScrollDirection)scrollDirection
   return CGRectExpandToRangeWithScrollableDirections(rect, tuningParameters, _scrollableDirections, scrollDirection);
 }
 
+- (ASRangeTuningParameters)tuningParametersForRangeType:(ASLayoutRangeType)rangeType
+{
+    BOOL isVisible = ASInterfaceStateIncludesVisible(_collectionView.interfaceState);
```

Comment:
Okay, now I see where this is used :-) maybe we could put this method definition in an internal header.